src/styles/blocks/button-card.scss

```diff
@@ -0,0 +1,27 @@
+/* #region Button card */
```

Comment:
Regions are good for large css files with lot of classes.
You really don't need them if you already divide styles one file per block.

Reply:
Thanks for the comments )